### ISSUE TITLE
download not working

### ISSUE THREAD

**Issue by tbleiker** [ThePromidius/tachiyomi-extensions#25]
### Source information

Kavita v1.3.5

### Steps to reproduce

try to download a resource from a kavita server

### Expected behavior

the resource is downloaded

### Actual behavior

Download is shown in the android notification bar. But the progress bar is stuck at 0%. After a while the download is shown as failed in the app, but as "finished" in the progress bar.

### Tachiyomi version

stable v0.13.5

### Android version

Adnroid 11

### Other details

- Kavita docker installation (v0.5.4) on debian server
- Books can be downloaded over OPDS with other apps (e.g. moon reader or librera)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by ThePromidius** [comment]
I just tested on 2 devices and had no issue.
Only thing i can think of is slow internet speed or something along that lines.
StilI, I doubt this is an issue with the extension itself.
If you can read those chapter properly without downloading then i'm sure its not related to the extension

**Comment by ThePromidius** [comment]
That's right since the new book reader it's not supported. Because it no longer process the pdfs as images.
It's on my todo-list to remove support from the extension
